examples/package.json

```
@@ -6,5 +6,6 @@
     "build": "pnpm --filter './*' --if-present build",
     "clean": "pnpx rimraf ./*/node_modules/ ./*/.vinxi/ ./*/.output/",
     "use-workspace-package": "node ./use-workspace-package-in-examples.js"
-  }
+  },
+  "packageManager": "pnpm@10.6.1"
```

```suggestion
  "packageManager": "pnpm@10.8.1"
```

10.6.1 isn't listed on https://github.com/pnpm/pnpm/releases - let's try again with latest!